src/main/java/com/codecool/virtualpub/logic/Pub.java

```diff
@@ -11,8 +13,10 @@ public class Pub {
     private List<Customer> customers;
     private Bartender bartender;
     private int profit;
+    private Display display = new Display();
```

Comment:
Getting Input and Display from outside of the class ensures that we can easily change them if we need another kind of Input or Display. Right now they are hard-coded which means we have to change this class if some changes are needed with the Display or Input